client/src/css/contactStyles.css

```diff
@@ -0,0 +1,231 @@
+  color: white;
+  border: none;
+  font-family: Nunito;
+  font-size: 20px;
+}
+
+/* Styling for send message page */
+
+.newMessage {
+  position: absolute;
+  width: min(1090px, 60vw);
+  height: 60vh;
+  left: 35vw;
+  top: 27vh;
+
+  background: #ffffff;
+  border: 1px solid #000000;
+  box-sizing: border-box;
+  box-shadow: 5px 5px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 15px;
+}
+
+.topBar {
+  position: absolute;
+  width: 100%;
+  height: 10vh;
+  font-weight: 700;
+  font-size: min(30px, max(20px, 2vw));
+  padding-left: 4%;
+  padding-right: 1%;
+  background: #8dc442;
+  box-sizing: border-box;
+  border-radius: 15px 15px 0px 0px;
+
+  color: #ffffff;
+  display: flex;
+  justify-content: space-between;
+}
+
+.cancel {
+  padding-right: 10px;
+  background: none;
+  border: none;
+}
+.cancelImg {
+  width: max(18px, min(30px, 3vw));
+}
+
+.selection2 {
+  border-radius: 10px;
+  position: relative;
+  width: max(250px, 20vw);
+  top: 10vh;
+  outline: none;
+  font-size: min(20px, 2vw);
+  color: #53575a;
+}
+
+.form {
+  position: absolute;
+  width: 96%;
+  height: 43vh;
+  bottom: 2vh;
+  left: 2%;
+  font-size: min(30px, max(20px, 2vw));
+}
+
+.errorMessage {
+  position: absolute;
+  top: 0;
+  padding: 0;
+  margin: 0;
+  color: red;
+}
+.inputArea {
+  width: 100%;
+  height: 90%;
+  position: absolute;
+  bottom: 0;
+
+  direction: ltr;
+  padding: 2%;
+  outline: none;
+  resize: none;
+  background: #ffffff;
+  border: 0.5px solid #9b9a9a;
+  box-sizing: border-box;
+  border-radius: 15px;
+}
+
+.sendButton {
+  background: #8dc442;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 20px;
+  border: none;
+
+  position: absolute;
+  right: 3.5vw;
+  bottom: 4vh;
+  float: right;
+  color: white;
+  width: min(15vw, 191px);
+  height: 5vh;
+  z-index: 1;
+  font-size: 20px;
+}
+
+.messageReceived {
+  position: absolute;
+  z-index: 1;
+  width: 30%;
+  height: 16vh;
+  left: 35%;
+  top: 45%;
+  background-color: white;
+
+  border: 5px solid #8dc442;
+  box-sizing: border-box;
+  border-radius: 20px;
+}
+.message {
+  width: 90%;
+  height: 100%;
+  padding: 0% 5%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  font-size: min(18px, 1.5vw);
+}
+
+.smallCancel {
+  border: none;
+  background: none;
+  position: absolute;
+  top: 5%;
+  left: 2%;
+}
+
+/* Mobile Views */
+
+@media screen and (max-width: 750px) {
+  .contactLanding {
+    width: 90%;
+    margin: 0% 5%;
+    padding: 5%;
+    left: 0;
+    top: 23%;
+  }
+
+  .helpQuestion {
+    font-size: 5vw;
+  }
+  .newMessage {
+    left: 0;
+    width: 90%;
+    margin: 0% 5%;
+    top: 15%;
+  }
+  .messageReceived {
+    width: 75%;
+    left: 12.5%;
+  }
+  .message {
+    font-size: max(13px, min(20px, 3vw));
+  }
+  .sendButton,
+  .continueButton {
+    width: min(100px, 50vw);
+    font-size: min(20px, 5vw);
+  }
+  .selection,
+  .selection2,
+  .inputArea {
+    font-size: min(10px, 4vw);
```

Comment:
On mobile views, the font is extremely small:

<img width="200" alt="image" src="https://user-images.githubusercontent.com/24444266/155641526-d45a99c0-fc06-40c8-884c-18acb2e5f904.png">

It looked better when I did this though:
```suggestion
    font-size: max(10px, 4vw);
```
(but you may have a better solution)